### PR TITLE
cve_update

### PR DESCRIPTION
**Pull request by pj-spoelders**

### CODE REVIEW

**Comment by pj-spoelders** [comment]
Closes #13

File: .github/workflows/release_droid_upload_github_release_assets.yml [hunk 1]
```diff
@@ -24,7 +24,9 @@ jobs:
       - name: Build with Maven skipping tests
         run: mvn --batch-mode clean verify -DskipTests
       - name: Generate sha256sum files
-        run: find target -maxdepth 1 -name *.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
```
Comment (kaklakariada) [inline .github/workflows/release_droid_upload_github_release_assets.yml:27]: Mark this and other generated files as generated in .gitattributes.
Example: https://github.com/exasol/s3-document-files-virtual-schema/blob/main/.gitattributes

File: doc/changes/changes_1.1.1.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Virtual Schema for document data in files on Azure Blob Storage 1.1.1, released 2022-09-28
+
+Code name: CVE Warnings
+
+## Summary
+
+Fix CVE Warnings + Dependency Updates
+
```
Comment (kaklakariada) [inline doc/changes/changes_1.1.1.md:8]: No ticket number?

File: pom.xml [hunk 3]
```diff
@@ -3,14 +3,14 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>azure-blob-storage-document-files-virtual-schema</artifactId>
-    <version>1.1.0</version>
+    <version>1.1.1</version>
     <name>Virtual Schema for document data in files on Azure Blob Storage</name>
     <description>Adapter for document data access from files from Azure Blob Storage.</description>
     <url>https://github.com/exasol/azure-blob-storage-document-files-virtual-schema/</url>
     <properties>
         <junit.version>5.8.2</junit.version>
```
Comment (kaklakariada) [inline pom.xml:11]:
```suggestion
        <junit.version>5.9.1</junit.version>
```

File: pom.xml [hunk 4]
```diff
@@ -175,13 +175,14 @@
                         <exclude>CVE-2022-24823</exclude>
                         <exclude>CVE-2022-2047</exclude>
                         <exclude>CVE-2022-30187</exclude>
+                        <exclude>CVE-2022-38752</exclude>
```
Comment (kaklakariada) [inline pom.xml:178]: Add a comment with the dependency and a reason for this exclusion

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: .github/workflows/release_droid_upload_github_release_assets.yml (hunk 1 above)
Comment (pj-spoelders) [reply .github/workflows/release_droid_upload_github_release_assets.yml:27]: Done

File: doc/changes/changes_1.1.1.md (hunk 2 above)
Comment (pj-spoelders) [reply doc/changes/changes_1.1.1.md:8]: It's in the PR, but I'm going to add it.

File: pom.xml (hunk 3 above)
Comment (pj-spoelders) [reply pom.xml:11]: k

File: pom.xml (hunk 4 above)
Comment (pj-spoelders) [reply pom.xml:178]: ok

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**